WebApiCoreSeed.WebApi/Middleware/AuthorizationMiddleware.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace WebApiCoreSeed.WebApi.Middleware
+{
+    /// <summary>
+    /// Middleware to handle global authorization policies
+    /// </summary>
+    public class AuthorizationMiddleware
```

Comment:
I like the idea to have some example that how the middlewares works in .NET core because this new feature replace the HTTP modules...